lams_build/build_base.xml

```diff
@@ -372,7 +372,8 @@
 	<!-- ====================================================================== -->
 	<target name="_sass.do.compile">
         <apply dest="${basedir}/build/css" executable="${sass_exec_file}" failonerror="true" verbose="true">
-        	<arg value="--no-source-map"/>
+<!--        	<arg value="&#45;&#45;no-source-map"/>-->
+					<arg value="--sourcemap=none"/>
```

Comment:
Please see this commit for explanation what this form of the parameter is the way to go
https://code.lamsfoundation.org/fisheye/changelog/lams-github?cs=6aeea5d482f2cddbd3b75c4c722de6caf7ab633f